src/examples/next/pages/api/require-session.js

```diff
@@ -1,9 +1,9 @@
-const { requireSession } = require('@clerk/clerk-sdk-node');
+import clerk, { requireSession } from '@clerk/clerk-sdk-node';
 
 function handler(req, res) {
     console.log('Session required');
```

Comment:
@yourtallness Can we fix or surpress these Github actions linting suggestions? They kind of get in the way during the PR review

Reply:
#truedat, will look into it